main/guides/js-programming/hardened-js.md

```diff
@@ -263,7 +263,7 @@ or not:
 
 In compartments used to load Agoric smart contracts,
 `globalThis` is hardened, following OCap discipline.
-These compartments have `console` and `assert` globals from [SES](./ses/).
+These compartments have `console` and `assert` globals from [Hardened JavaScript](https://github.com/endojs/endo/blob/master/packages/ses/README.md).
```

Comment:
We're in the/a hardened js guide already. How about "the `ses` package" for the link text?